test/models/system_variable_test.exs

```diff
@@ -39,6 +41,70 @@ defmodule CoursePlanner.SystemVariableTest do
       changeset = SystemVariable.changeset(%SystemVariable{}, %{@string_valid_attrs | value: ""})
       refute changeset.valid?
     end
+
+    test "changeset fails when input is empty but field is not required" do
+      changeset = SystemVariable.changeset(%SystemVariable{}, %{@string_valid_attrs | value: "", required: false})
+      assert changeset.valid?
```

Comment:
sorry this test should pass, cause the field is not required. i'll update the test name